tests/join_test.py

```diff
@@ -1,88 +1,65 @@
-#!/usr/bin/env python3                                                         
-
 import importlib
```

Comment:
Several of the join tests fail for me when running with multiple nodes/locales. Running with `arkouda_server -nl 2`:

```
=================================================== short test summary info ====================================================
FAILED tests/join_test.py::JoinTest::test_join_on_eq_with_abs_dt - AssertionError: 1000 != 500
FAILED tests/join_test.py::JoinTest::test_join_on_eq_with_pos_dt - AssertionError: 1000 != 500
FAILED tests/join_test.py::JoinTest::test_join_on_eq_with_true_dt - AssertionError: 1000 != 500
================================================= 3 failed, 3 passed in 12.65s =================================================
```

Reply:
Odd, does that happen on the master branch also?

Reply:
Yeah, appears to fail on master too

Reply:
cc'ing @mhmerrill for join_test.py failures